styles/styles.css

```diff
@@ -15,6 +15,34 @@ header {
   box-shadow: 15px -15px #be7f6a;
 }
 
+.card {
+  background-color: #87adcf;
+  text-align: center;
+  margin: 3px;
+  padding: 1rem;
+}
+
+.question {
+  background-color: #87adcf;
+  text-align: center;
+}
+
+.button-answer {
+  color: #f3ead9;
+  background-color: #29282a;
+  border: 2px solid #e6664e;
+  padding: 1rem;
+}
```

Comment:
@PReichetanz there were some comments on here which got overwritten unfortunately when I pushed again. You weren't happy about overwriting the color of the button and that I took the same padding as in `.card` I guess. Other comments got deleted as well, now it's a mess :(